Slim/CallableResolver.php

```
@@ -182,8 +182,11 @@ private function bindToContainer(callable $callable): callable
             $callable = $callable[0];
         }
         if ($this->container && $callable instanceof Closure) {
-            /** @var Closure $callable */
-            $callable = $callable->bindTo($this->container);
+            /** @var Closure $bound */
+            $bound = @$callable->bindTo($this->container);
```

We don't suppress errors in this codebase. This isn't a fix.

---

I know. But there is no legal way to check that Closure is static.